test/test_instructions.py

```diff
@@ -52,6 +52,39 @@
     ZEBRA_ACCOUNTS,
 )
 
+def disable_hwm(snap_path: Path, backend: BackendInterface, firmware: Firmware, tezos_navigator: TezosNavigator) -> None:
+
+    if firmware.is_nano:
```

Comment:
Missing description here.
Can be moved in `TezosNavigator` as it's a navigation helper.

Reply:
Done.